src/zenml/integrations/feast/feature_stores/feast_feature_store.py

```diff
@@ -0,0 +1,104 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from typing import Any, ClassVar, Dict, List, Union
+
+import redis  # type: ignore
+from feast import FeatureStore  # type: ignore[import]
+from feast.feature_service import FeatureService  # type: ignore[import]
+from pandas import DataFrame
+
+from zenml.feature_stores.base_feature_store import BaseFeatureStore
+from zenml.logger import get_logger
+from zenml.stack.stack_component_class_registry import (
+    register_stack_component_class,
+)
+
+logger = get_logger(__name__)
+
+
+@register_stack_component_class
+class FeastFeatureStore(BaseFeatureStore):
+    """Class to interact with the Feast feature store."""
+
+    FLAVOR: ClassVar[str] = "feast"
+
+    online_host: str = "localhost"
+    online_port: int = 6379
+    feast_repo: str
```

Comment:
Is there more base configuration than just these things in Feast? How does one connect to a remote repo?

Reply:
I don't think that there is the concept of a 'remote' feature repository in Feast. You can store it separately as a separate repository on Github etc, but in order to be able to run the code you'll need to have a local version of it somewhere. See [here](https://docs.feast.dev/reference/feature-repository) for more on that.